components/hab/src/command/pkg/env.rs

```diff
@@ -12,18 +12,56 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::collections::HashMap;
 use std::path::Path;
 
 use hcore::package::{PackageIdent, PackageInstall};
 
 use error::Result;
 
-// TODO: This needs a windows compatible version
 pub fn start(ident: &PackageIdent, fs_root_path: &Path) -> Result<()> {
     let pkg_install = PackageInstall::load(ident, Some(fs_root_path))?;
     let env = pkg_install.environment_for_command()?;
+    render_environemt(env)?;
+    Ok(())
+}
+
+#[cfg(not(windows))]
+fn render_environemt(env: HashMap<String, String>) -> Result<()> {
     for (key, value) in env.into_iter() {
         println!("export {}=\"{}\"", key, value);
     }
     Ok(())
 }
+
+#[cfg(windows)]
+fn render_environemt(env: HashMap<String, String>) -> Result<()> {
+    use hcore::fs::FS_ROOT_PATH;
+    use std::env;
+    use std::path::PathBuf;
+
+    let mut paths: Vec<PathBuf> = match env.get("PATH") {
+        Some(path) => env::split_paths(&path).collect(),
+        None => vec![],
+    };
+    for i in 0..paths.len() {
+        if paths[i].starts_with("/") {
+            paths[i] = Path::new(&*FS_ROOT_PATH).join(paths[i].strip_prefix("/").unwrap());
+        }
+    }
+    let joined = env::join_paths(paths)?;
+    let mut mut_env = env.clone();
+
+    mut_env.insert(
+        String::from("PATH"),
+        joined
+            .into_string()
+            .expect("Unable to convert OsStr path to string!"),
+    );
+
+    for (key, value) in mut_env.into_iter() {
```

Comment:
Since this is just to print the values, rather than cloning the `env` argument and inserting, you could:
```rust
    let joined = &env::join_paths(paths)?
        .into_string()
        .expect("Unable to convert OsStr path to string!");

    for (key, value) in env.iter() {
        println!(
            "$env:{}=\"{}\"",
            key,
            if key == "PATH" { joined } else { value }
        );
    }
```